classes/Tab.php

```diff
@@ -304,7 +304,7 @@ public static function getModuleTabList()
      *
      * @return array tabs
      */
-    public static $_cache_tabs = [];
+    protected static $_cache_tabs = [];
```

Comment:
This is a BC

Reply:
It was already protected before changes. It just an error in my commit, finally if you check the changes, there is no change on this property